src/stores/annotations.ts

```diff
@@ -314,8 +318,8 @@ export const useAnnotationsStore = defineStore('annotations', () => {
       if (!target.dataset.annotation) {
         target = getNearestParentAnnotation(target);
       }
-
-      if (!target) {
+      
+      if (!target ||  target.getAttribute('data-annotation-level') < 0) {
```

Comment:
We identified today that this wasn't really the issue of the bug. So this is not needed anymore I guess.

Reply:
yes, true.